packages/cli-lib/lib/text.js

```diff
@@ -8,6 +10,20 @@ const commaSeparatedValues = (arr, conjunction = 'and', ifempty = '') => {
   return arr.join(', ');
 };
 
+/**
+ * These helper methods are used to modify text output within the CLI. They
+ * should all take in a string value and output a modified string value.a
+ */
+const helpers = {
```

Comment:
This might be better off living in the interpolation file. I can imagine someone adding extra options to this object and using it outside of the interpolation context. Every new key added here results in another call to `compileHelper` in the interpolation file.

Reply:
I originally had this in a separate `interpolationHelpers` file, but after looking at it, I believe this could be used more generally to modify any other text in the CLI.